components/Dialogs/src/MDCAlertController.m

```diff
@@ -62,7 +62,7 @@ - (id)copyWithZone:(NSZone *)zone {
 static const CGFloat MDCDialogActionButtonHeight = 36.0;
 static const CGFloat MDCDialogActionButtonMinimumWidth = 48.0;
 
-static const CGFloat MDCDialogMessageOpacity = 0.38f;
+static const CGFloat MDCDialogMessageOpacity = 0.6f;
```

Comment:
Discussed offline, but for posterity: the [dialogs spec](https://material.io/guidelines/components/dialogs.html#dialogs-alerts) seems to be using [primary opacity (87%) for the title and secondary (54%) for the body text](https://material.io/guidelines/style/color.html#color-usability).